lib/src/dev/mt_dev.c

```diff
@@ -1598,16 +1666,15 @@ struct mt_tx_queue* mt_dev_get_tx_queue(struct mtl_main_impl* impl, enum mtl_por
       if (ret < 0) {
         err("%s(%d), fallback to tsc as rl fail\n", __func__, port);
         inf->tx_pacing_way = ST21_TX_PACING_WAY_TSC;
+      } else {
+        bps_g = (float)tx_queue->bps * 8 / (1000 * 1000 * 1000);
```

Comment:
we have a define that does seconds ? 
Like NS_PER_S 
usually in this codespace ?

Reply:
We do have NS_PER_S which is equal to (1000 * 1000 * 1000) but here this value is used to make bytes from gigabytes, no NS from S. I cloud define B_from_GB , but... I guess it is self explanatory enough